examples/front-proxy/Dockerfile-service

```diff
@@ -6,4 +6,4 @@ RUN mkdir /code
 ADD ./service.py /code
 ADD ./start_service.sh /usr/local/bin/start_service.sh
 RUN chmod u+x /usr/local/bin/start_service.sh
-ENTRYPOINT /usr/local/bin/start_service.sh
+ENTRYPOINT /bin/bash /usr/local/bin/start_service.sh
```

Comment:
You should change the shebang to bash in the script instead.

Reply:
I didn't see the shebang, I changed the entrypoint to /bin/sh

Reply:
https://github.com/envoyproxy/envoy/blob/master/examples/front-proxy/start_service.sh#L1 This line?

Reply:
yes

Reply:
So why did you need this change?

Reply:
I get the following error:
docker logs -f a342aa41445bb0d63298f1bd695d1ab12cf9bec99b5ce35cc79c2fbdb98c15f6
/bin/sh: /usr/local/bin/start_service.sh: not found